specs/segragate_object_specific_flows.adoc

```diff
@@ -0,0 +1,295 @@
+===============================
+Segregate object specific flows
```

Comment:
Are we following ascii doc formatting here.
Headings should start with "=", "==" etc..